src/compas/geometry/_core/angles.py

```diff
@@ -352,6 +352,30 @@ def angles_points_xy(a, b, c, deg=False):
     v = subtract_vectors_xy(c, a)
     return angles_vectors_xy(u, v, deg)
 
+def angle_planes(a, b, deg=False):
+    r""" Compute the smallest angle between the two normal vectors of two planes.
+
+    Parameters
+    ----------
+    a :  :class:`compas.geometry.Plane`
+    b :  :class:`compas.geometry.Plane`
+    deg : boolean
+        returns angles in degrees if True
+
+    Returns
+    -------
+    float
+        The smallest angle in radians (in degrees if deg == True).
+        The angle is always positive.
+
+    Examples
+    --------
+    >>> angle_planes(plane_a, plane_b, True)
```

Comment:
the example would be more meaningful (and doctest-able) if you actually compute the angle between two planes, such as XY and YZ and output the result...

Reply:
yes, you are right, i will incorporate that